posts/customer-360.md

```diff
@@ -0,0 +1,37 @@
+
+Now, where do we start, and what are the steps we must take to successfully implement this?
+
+![Customer 360 Visual](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/08/Customer360.png)
+
+## Data Discovery & Inventory
+First, we must identify the sources of customer data we’ll be relying on. Perhaps your company already manages this type of activity well: you have an enterprise data warehouse, good documentation, maybe even a robust data catalog. Likely, that is not the case. This first step involves developing an inventory of data sources and analyzing the complexity of obtaining that data. Profiling the data to understand inconsistencies will save time reconciling quality issues down the road and possibly even yield some short term wins by identifying data quality patterns that can quickly be resolved upstream.
+
+Within different departments and lines of business there will undoubtedly be “data gurus”- those people who really understand the data and its quirks. Identifying these people and developing a relationship with them is key to truly documenting and inventorying this data (fun side note- this could be a great jumping off point for a Data Catalog or data dictionary!). While the author of this blog is clearly biased; a third party consulting firm is most useful at this stage. The ability to see the forest from the trees and bring outside perspective provides a huge advantage.
+
+## Data Hygiene & Integration
+Once we’ve done the work of finding out what data we need, where it resides, and what issues we are likely to encounter with it, comes the arduous task of improving the data’s hygiene. Data Quality issues are rife in organizations; it’s just the nature of the beast. Outdated systems, poor processes, messy mergers, third party data, etc. all leave us with some degree of data hygiene problems. Oceans of digital ink have been spilled on the topic of Data Quality, but suffice it to say that prioritizing the issues that are most likely to diminish the impact of a Customer 360 is a great place to start. In a perfect world, all of these issues will be resolved at their source; but that isn’t always possible and so transformations will be required. This will be an ongoing effort so a thoughtful, well documented approach will serve the future of the business well.
```

Comment:
```suggestion
Once we’ve done the work of finding out what data we need, where it resides, and what issues we are likely to encounter with it, comes the arduous task of improving the data’s hygiene. Data Quality issues are rife in organizations -- it’s just the nature of the beast. Outdated systems, poor processes, messy mergers, third party data, etc. all leave us with some degree of data hygiene problems. Oceans of digital ink have been spilled on the topic of Data Quality, but suffice to say, prioritizing the issues that are most likely to diminish the impact of a Customer 360 is a great place to start. In a perfect world, all of these issues will be resolved at their source, but since that isn’t always possible, transformations will be required. This will be an ongoing effort, so a thoughtful, well-documented approach will serve the future of the business well.
```